lib/round-tracker.js

```diff
@@ -57,15 +57,22 @@ export async function startRoundTracker (pgPool) {
     }
   }
 
-  contract.on('RoundStart', (newRoundIndex) => {
+  const onRoundStart = (newRoundIndex) => {
     updateSparkRound(newRoundIndex).catch(err => {
       console.error('Cannot handle RoundStart:', err)
       Sentry.captureException(err)
     })
-  })
-
-  const currentRound = await updateSparkRound(await contract.currentRoundIndex())
-  return currentRound
+  }
+  contract.on('RoundStart', onRoundStart)
+  const close = () => contract.removeListener('RoundStart', onRoundStart)
```

Comment:
I need this in order to allow the test to stop the tracker (unsubscribe from the events) after the test is done.